app/src/main/AndroidManifest.xml

```diff
@@ -22,6 +22,15 @@
         android:label="@string/APP_NAME"
         android:largeHeap="true"
         android:theme="@style/Hedvig.Theme">
+        <activity android:name=".feature.onbarding.ui.MoreOptionsActivity"></activity>
+        <activity
+            android:name=".feature.onbarding.ui.ChoosePlanActivity"
+            android:configChanges="keyboard|keyboardHidden|orientation|screenSize"
+            android:launchMode="singleTask"
+            android:parentActivityName=".feature.marketing.ui.MarketingActivity"
```

Comment:
```suggestion
```

I'm unfamiliar with this attribute, should we really use it?

Reply:
It is to make the back navigation in the toolbar go to this screen. I later changed my mind and dit it the way we use too but forgot to remove this line